### PR TITLE
Cross-build for Scala 2.13

### PR DESCRIPTION
**Pull request by armanbilge**
Closes https://github.com/VirtusLab/scala-yaml/issues/159.

I had a brief look at adding 2.12 as well, but that still requires more changes. 2.13 is pretty good for now :)

### CODE REVIEW

**Comment by kpodsiad** [comment]
Seems like this PR is on good way @armanbilge ;)

File: build.sbt [hunk 1]
```diff
@@ -36,13 +37,11 @@ inThisBuild(
 
 ThisBuild / scalafixDependencies += "com.github.liancheng" %% "organize-imports" % "0.6.0"
 
-lazy val core = crossProject(JSPlatform, JVMPlatform, NativePlatform)
+lazy val core = crossProject(JSPlatform, JVMPlatform /*, NativePlatform*/ )
```
Comment (kpodsiad) [inline build.sbt:40]: Is there a reason for commenting native version?

**Review by kpodsiad — COMMENTED** [review]
scalafmt and native fails on CI, after they're green I think this will be good to go 👍

File: build.sbt (hunk 1 above)
Comment (armanbilge) [reply build.sbt:40]: Yes, because I had to bump to Scala 3.2.0-RC2 the Native plugin is not yet published for these versions.

Comment (kpodsiad) [reply build.sbt:40]: CI for for native can be disabled (for a while) to not block this PR

Comment (armanbilge) [reply build.sbt:40]: @kpodsiad cool, thanks, I've fixed formatting and disabled Native CI for now.

**Review by kpodsiad — APPROVED** [review]
Looks good, CI is green 👍

**Pull request merged**